cuda/CMakeLists.txt

```diff
@@ -18,17 +18,13 @@ set(SOURCES
     stop/residual_norm_reduction_kernels.cu)
 
 enable_language(CUDA)
-include(../cmake/detect_gpu.cmake)
+include(${CAS_SOURCE})
```

Comment:
I guess this comes from CAS? This feels a bit confusing due to the way CMake works.

Reply:
From CAS's CMakeLists.txt, which is automatically added to the project by `load_git_repository`.

I could do an alternative: update the `CMAKE_MODULE_PATH` in CAS's `CMakeLists.txt`, but then I also need to propagate the module path to the parent in:

1. third_party/CudaArchitectureSelector/CMakeLists.txt
2. third_party/CMakeLists.txt

Then, this inlcude can be replaced with `include(CudaArchitectureSelector)`.

It is more code, but maybe it's easier to understand and a bit more standard way of doing things.

Reply:
That is maybe a good idea actually, as it is essentially a CMake module and this feels like the standard way of manipulating them.